src/domains/buildService/makeBuildResource.ts

```diff
@@ -0,0 +1,139 @@
+import { spawn } from "child_process";
+
+import { FrameWorkPresets } from "@src/domains/@constants/frameWorkPresets";
+import log from "@src/common/Logger";
+
+type Options = {
+  repoCloneUrl: string;
+  repoName: string;
+  framework: string;
+  installCommand: string;
+  buildCommand: string;
+};
+
+const runGitClone = async ({ repoCloneUrl }: Pick<Options, "repoCloneUrl">) => {
+  const command = [
+    `rm -rf buildResource`,
+    `mkdir buildResource`,
+    `cd buildResource`,
+    `git clone ${repoCloneUrl}`,
+  ].join(" && ");
+
+  const childProcess = spawn(command, {
+    cwd: "./",
+    shell: true,
+    stdio: "pipe",
+  });
+
+  childProcess.stdout.setEncoding("utf8");
+  childProcess.stderr.setEncoding("utf8");
+  childProcess.stdout?.on("data", data => {
```

Comment:
```suggestion
  childProcess.stdout?.on("data", log.debug);
```

이렇게 작성되면 더 깔끔할 것 같습니당~~

Reply:
oh.. 너무 깔끔해졌네요👍